app/services/builders/blog_metric_chart/growth_month_over_month.rb

```diff
@@ -0,0 +1,27 @@
+module Builders
+  module BlogMetricChart
+    class GrowthMonthOverMonth < Builders::BlogMetricChart::Base
+      def initialize(months = 13)
+        super(months + 1)
+      end
+
+      private
+
+      def process_data(metrics_data)
+        metrics_data.each.inject({}) do |hash, key_value_pair|
+          month, metric_value = key_value_pair
+          previous_metric_value = metrics_data[month.last_month]
+          next(hash) if previous_metric_value.nil? || previous_metric_value.zero?
+
+          hash.merge!(
+            month => growth_for(metric_value, previous_metric_value)
+          )
```

Comment:
Maybe this could be written as... 

```suggestion
          hash[month] = growth_for(metric_value, previous_metric_value)
```

WDYT?

Reply:
Oh, right, I completely forgot I could write it that way. Love it!